pybliometrics/scopus/superclasses/search.py

```diff
@@ -60,9 +60,18 @@ def __init__(self, query, api, refresh, view='STANDARD', count=200,
         ValueError
             If the api parameter is an invalid entry.
         """
-        fname = md5(query.encode('utf8')).hexdigest()
+        params = {'count': count, 'view': view}
+        if api != 'SerialSearch':
+            name = query
+            params['query'] = query
+        else:
+            name = ''
+            params.update(query)
+            for q in query:
+                name += q + ' ' + query[q] + '&'
+            name = name[:-1]
```

Comment:
Replace by `name = "&".join(["=".join(t) for t in zip(query.keys(), query.values())])`. Faster and easier to read